mod/01_rating/set/type/metric_value.rb

```diff
@@ -25,32 +25,34 @@ def source_exist?
   text_card = subfield :text
   link_card = subfield :wikirate_link
 
-  ( file_card && file_card.attachment.present? ) || 
-  ( text_card && text_card.content.present? ) ||
-  ( link_card && link_card.content.present? )
-
+  (file_card && file_card.attachment.present?) ||
+    (text_card && text_card.content.present?) ||
+    (link_card && link_card.content.present?)
 end
 
-event :set_metric_value_name, :before=>:set_autoname, :when=>proc{|c| c.cardname.parts.size < 4} do
+event :set_metric_value_name,
+      before: :set_autoname, when: proc { |c| c.cardname.parts.size < 4 } do
   self.name = ['metric', 'company', 'year'].map do |name|
-      content = remove_subfield(name).content
-      content.gsub('[[','').gsub(']]','')
-    end.join '+'
+    content = remove_subfield(name).content
+    content.gsub('[[', '').gsub(']]', '')
+  end.join '+'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.